src/MicrosoftTeams.schema.json

```diff
@@ -173,36 +172,107 @@
+                                        "properties": {
+                                            "title": {
+                                                "type": "string",
+                                                "description": "The one word command name",
+                                                "maxLength": 16
+                                            },
+                                            "usageHint": {
+                                                "type": "string",
+                                                "description": "A short usage template for the command",
+                                                "maxLength": 32
+                                            },
+                                            "shortDescription": {
+                                                "type": "string",
+                                                "description": "A short description of the command",
+                                                "maxLength": 32
+                                            },
+                                            "value": {
+                                                "type": "string",
+                                                "description": "Payload to use for the bot action type. E.g, a URL for an OpenURL type."
+                                            },
+                                            "type": {
+                                                "type": "string",
+                                                "enum" : ["imBack", "invoke", "openURL", "prompt", "gotoTab"],
+                                                "description": "The type of bot interaction the command defines.",
+                                                "maxLength": 16
+                                            },
+                                            "image": {
+                                                "type": "string",
+                                                "description": "An optional URL to an image to use for the command entry"
+                                            }
+                                        },
+                                        "required":[
+                                            "title",
+                                            "value",
+                                            "type"
+                                        ]
+                                    }
+                                }
+                            },
+                            "required":[
+                                "scope",
+                                "commands"
+                            ]
                         }
                     }
                 },
                 "required": [
                     "botId",
-                    "scope"
+                    "scope",
+                    "needsChannelSelctor"
```

Comment:
The "needsChannelSelctor" property shouldn't be required; if not specified we will just assume it's false.